llvm/test/CodeGen/AMDGPU/hazards-gfx950.mir

```diff
@@ -485,6 +485,23 @@ body:             |
     S_SETPC_B64_return undef $sgpr30_sgpr31, implicit killed $vgpr0
 ...
 
+---
+# GCN-LABEL: test_scalef32_sr_pk_fp4_bf16_opsel0_hazard
+# GCN:      V_CVT_SCALEF32_SR_PK_FP4_BF16_e64
```

Comment:
I don't know why this test isn't using generated checks, but this should use -NEXT